content/blog/usethis-2-0-0/index.md

```diff
@@ -168,83 +163,68 @@ When usethis configures a new Git remote, it must choose a protocol, either HTTP
 <div class="highlight">
 
 <pre class='chroma'><code class='language-r' data-lang='r'><span class='nf'><a href='https://usethis.r-lib.org/reference/git_protocol.html'>git_protocol</a></span><span class='o'>(</span><span class='o'>)</span>
-
 <span class='c'>#&gt; <span style='color: #BBBB00;'>ℹ</span><span> Defaulting to https Git protocol</span></span>
-
-<span class='c'>#&gt; [1] "https"</span>
-</code></pre>
+<span class='c'>#&gt; [1] "https"</span></code></pre>
 
 </div>
 
 As always, you can specify the default protocol for a single session with [`use_git_protocol()`](https://usethis.r-lib.org/reference/git_protocol.html) or for all sessions via the `usethis.protocol` option. Those who prefer SSH may want to set this option in `.Rprofile` going forward.
 
-Pull request helpers
---------------------
+## Pull request helpers
 
 The team that maintains the tidyverse and r-lib packages makes heavy use of GitHub pull requests for managing internal and external contributions[^3]. The `pr_*()` family of functions supports pull request workflows, for maintainers and contributors. This family has gained a couple of new functions and some improvements to existing functions:
 
 -   [`pr_resume()`](https://usethis.r-lib.org/reference/pull-requests.html) resumes work on an existing local PR branch. It can be called argument-less, to select a branch interactively.
 
-    <div class="highlight">
-
-    <pre class='chroma'><code class='language-r' data-lang='r'>> pr_resume()
-      ℹ No branch specified ... looking up local branches and associated PRs
-      Which branch do you want to checkout? (0 to exit) 
-
-      1: avalcarcel9-add_use_author --> #833 ('@avalcarcel9'): Add use author
-      2:                 latex-hell
-      3:                     holder
-      4:            patch-for-withr
+        > pr_resume()
+        ℹ No branch specified ... looking up local branches and associated PRs
+        Which branch do you want to checkout? (0 to exit) 
```

Comment:
Yeah, I think we lost some highlighting here.

Reply:
I don't see it on https://www.tidyverse.org/blog/2020/12/usethis-2-0-0/#pull-request-helpers ? (And the indent there looks wacky).